.nvmrc

```diff
@@ -0,0 +1 @@
+v10.15.3
```

Comment:
why is this file necessary?

Reply:
most people have multiple versions of node installed at any given moment using nvm... when there's an `nvmrc` file you can just type in `nvm use` and it will set the expected version for you.  it's like a cue for people as to what version of node they should be using for this project.  can't hurt to be explicit about it...

Reply:
fair enough...